type/__packages/manifest

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh -e
+
+os=$(cat "${__global}/explorer/os")
+
+___package() {
```

Comment:
Please rename this function, it's very confusing.
At least it confused me when I read the code and thought you were calling the `__package` type (I know it has more underscores, but that's easily overlooked).

Maybe `do_package`, `process_package` ?